shared-kernel-infrastructure/src/main/java/org/dcsa/skernel/infrastructure/sorting/Sorter.java

```diff
@@ -0,0 +1,65 @@
+package org.dcsa.skernel.infrastructure.sorting;
+
+import lombok.extern.slf4j.Slf4j;
+import org.dcsa.skernel.errors.exceptions.ConcreteRequestErrorMessageException;
+import org.dcsa.skernel.infrastructure.pagination.Cursor;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+
+/** Helper class for transforming the DCSA Sorting syntax to Spring-data Sorting. */
+@Slf4j
```

Comment:
We do not seem to be using the logging, so the annotation looks like it is unused.

Reply:
annotation removed